src/migrations/1623310575144-agent-auth.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class agentAuth1623310575144 implements MigrationInterface {
```

Comment:
Do we need two migrations for one story?

Reply:
0373ff34429161e383c18c9ee5111b08a1a8f27d